### ISSUE TITLE
Installed Pop!_OS in a VM with VMWare Fusion and couldn't get Firefox to open

### ISSUE THREAD

**Issue by tiomimbre** [system76/pop-distro#165]
If this is a bug, please use the template below. If this is a feature request, question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**

```
 * Distribution - (run ```cat /etc/os-release```)
 * Related Application and/or Package Version - apt policy $PACKAGE NAME
```
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 17.10 (Artful Aardvark)"
VERSION_ID="17.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com/"
BUG_REPORT_URL="https://github.com/system76/pop-distro/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful

**Issue/Bug Description**
Installed Pop!_OS in a VM with VMWare Fusion and couldn't get Firefox to open

**Steps to reproduce (if you know)**
Install Pop OS in a VM with VMware Fusion, install VMware tools and open firefox.

**Expected behaviour**

**Other Notes**
Syslog: 
pop-os kernel: [ 3067.880811] firefox[5461]: segfault at ffffffffffffffff ip 00007f2692651100 sp 00007ffd12a1fe18 error 7 in libxul.so[7f268dfc9000+578e000]

**Comment by brs17** [comment]
@tiomimbre So I do not have the ability to run VMWare Fusion since I do not have a Mac at my disposal, but I have tested Pop!_OS in both Gnome Boxes as well as Oracle Virtualbox and was able to run Firefox just fine. 

@jackpot51 does that segfault mean anything to you?

**Comment by jackpot51** [comment]
@tiomimbre how much memory did you have in the VM ?

**Comment by tiomimbre** [comment]
2 GB of memory

**Comment by tiomimbre** [comment]
I'm going to try removing and installing firefox again.

**Comment by brs17** [comment]
Are you able to open other programs in Pop!_OS?

**Comment by tiomimbre** [comment]
yeah, I can open other programs including chromium. I just removed firefox from the command line: apt remove firefox and install it again with apt install firefox and now Firefox is working fine.

**Comment by jackpot51** [comment]
Due to firefox working after reinstall, and not being able to reproduce the problem, we will be closing this issue. Feel free to reopen if you experience the problem again.
